common/config/environment.go

```diff
@@ -46,15 +46,15 @@ var (
 	once     sync.Once
 )
 
-// GetEnvInstance ...
+// GetEnvInstance get env instance by singleton
```

Comment:
get -> gets? and the following errors should be fixed too.

Reply:
so, find should be finds, update should be updates and so no

Reply:
ok